sample-apps/radiology/lib/configs/segmentation_spleen.py

```diff
@@ -55,9 +55,13 @@ def init(self, name: str, model_dir: str, conf: Dict[str, str], planner: Any, **
         # Download PreTrained Model
         if strtobool(self.conf.get("use_pretrained_model", "true")):
             url = f"{self.conf.get('pretrained_path', self.PRE_TRAINED_PATH)}"
-            url = f"{url}/radiology_segmentation_unet_spleen.pt"
+            url = f"{url}/radiology_segmentation_unet_spleen_TotalSeg.pt"
```

Comment:
can u make the file name lower case.. just to be sync with others..

Reply:
fully agree. Just changed it